ui/components/providers.js

```diff
@@ -48,9 +48,22 @@ function oidc(id, clientID, authURL, scopes, redirectURL, next) {
     .join('')
   window.localStorage.setItem('state', state)
 
-  document.location.href = `${authURL}?redirect_uri=${redirectURL}&client_id=${clientID}&response_type=code&scope=${scopes.join(
-    '+'
-  )}&state=${state}`
+  const sendTo = new URL(authURL)
+  // URL searchParams add query parameters to a URL
+  sendTo.searchParams.append('redirect_uri', redirectURL)
+  sendTo.searchParams.append('client_id', clientID)
+  sendTo.searchParams.append('response_type', 'code')
+  sendTo.searchParams.append('scope', scopes.join(' '))
+  sendTo.searchParams.append('state', state)
+
+  if (kind === 'google') {
```

Comment:
Is there any harm in _always_ adding these params?

Reply:
In the case of Azure it means an extra step approving permissions every time you login, so I'd like to keep it specific for now.

Reply:
Thanks!